src/fides/api/api/v1/endpoints/privacy_preference_endpoints.py

```diff
@@ -187,6 +188,26 @@ def get_request_origin_and_config() -> Tuple[Optional[str], Optional[str]]:
     )
 
 
+def get_ip_address(request: Request) -> Optional[str]:
+    """Get client ip, preferring x-forwarded-for if it exists, otherwise, dropping back to
+    request.client.host"""
+    x_forwarded_for = (
+        request.headers.get("x-forwarded-for") if request.headers else None
```

Comment:
I am not sure if I need to separately allow `forwarded-allow-ips` to be configured as an env var to pass into uvicorn 

https://www.uvicorn.org/settings/#http

> --proxy-headers / --no-proxy-headers - Enable/Disable X-Forwarded-Proto, X-Forwarded-For, X-Forwarded-Port to populate remote address info. Defaults to enabled, but is restricted to only trusting connecting IPs in the forwarded-allow-ips configuration.

> --forwarded-allow-ips Comma separated list of IPs to trust with proxy headers. Defaults to the $FORWARDED_ALLOW_IPS environment variable if available, or '127.0.0.1'. A wildcard '*' means always trust.

Reply:
interesting, is that saying the _header_ is rejected if that's not configured, or the request in general? (this is probably obvious and i'm just missing context!)

seems unlikely we're configuring this already (i don't see any reference to the config keys anywhere in either fides/fidesplus/fidesinfra codebases)...but i'm just spitballing!

Reply:
I can't tell from the docs! I am not sure if it's needed, I tested this alpha tag https://github.com/ethyca/fides/releases/tag/2.24.2a0 on a staging environment Robert set up, and it looks like X-Forwarded-For is getting passed through to the application just fine?